devops/databases/sql_snippets_postgres_script.rst

```diff
@@ -0,0 +1,100 @@
+.. _commandsscripts:
+
+============================
+Commands Scripts (Postgress)
+============================
+
+
+**Some scripts are available to facilitate various tasks**
```

Comment:
~~**Some scripts are available to facilitate various tasks**~~

I'd avoid any text that doesn't add real meaning.